lib/tasks/test.js

```diff
@@ -264,5 +270,46 @@ module.exports.browserTest = function(gulp, config) {
 	});
 };
 
+module.exports.pa11yTest = function(gulp, config) {
+	log.secondary('Running "pa11y test"...');
+	config = config || {};
+	const defaultSrc = path.join(process.cwd(), '/demos/local/pa11y.html');
+	// ignoring by default warnings, notices and the href="#" error
+	const defaultIgnore = ['WCAG2AA.Principle2.Guideline2_4.2_4_1.G1,G123,G124.NoSuchID', 'warning', 'notice'];
+	let src = (config.pa11yPath !== undefined) ? path.join(process.cwd(), config.pa11yPath) : defaultSrc;
+	let ignore = (config.pa11yIgnore !== undefined) ? config.pa11yIgnore.split(';') : defaultIgnore;
+
+	return new Promise(function(resolve, reject) {
+		fs.readFile(src, function(err) {
+			if (err === null) {
+				const pa11y = require('pa11y');
+				const pa11yCliReporter = require('pa11y/reporter/cli');
+
+				const test = pa11y({
+					log: {
+						debug: console.log.bind(console),
+						error: console.error.bind(console),
+						info: console.info.bind(console)
+					},
+					ignore: ignore
+				});
+
+				src = 'file://' + src;
```

Comment:
I think this might restrict us at some point. Could we only add "file://" if a protocol isn't already set? E.g.

```
foo/bar          becomes  file://foo/bar
file://foo/bar   becomes  file://foo/bar
http://example/  becomes  http://example/